src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_footer/overview_page_footer.tsx

```diff
@@ -91,7 +96,12 @@ export const OverviewPageFooter: FC<Props> = ({ addBasePath, path }) => {
       >
         <FormattedMessage
           id="opensearch-dashboards-react.pageFooter.makeDefaultRouteLink"
-          defaultMessage="Make this my landing page"
+          defaultMessage={i18n.translate(
+            'opensearch-dashboards-react.pageFooter.makeDefaultRouteLink',
```

Comment:
It doesn't seem right that we'd need to duplicate the string id in both the `id` prop and the `defaultMessage`. My naive understanding from a quick skim of the code is that `<FormattedMessage>` is a component provided by `osd-i18n` package, and all it needs to work correctly is the `<I18nProvider>` component to provide the context: https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/packages/osd-i18n#react

Reply:
good point. I think to get it work by using `<I18nProvider>` we need to get a mount element when it renders by calling `ReactDOM.render`. In our case, we don't have such params.

```
export const renderApp = ({ element, history }: AppMountParameters, { basePath }: Deps) => {
  ReactDOM.render(
    <I18nProvider>
      <ErrorApp history={history} basePath={basePath} />
    </I18nProvider>,
    element
  );
```

To solve duplication, since both `i18n.translate` and `<FormattedMessage>` can be used to translate strings. I am thinking we could replace  many un-functional `<FormattedMessage>` with i18n.translate (see below code snippet). Below also works well:

```
const defaultRoutebutton =
    defaultRoute === path ? (
      <RedirectAppLinks application={application}>
        <EuiButtonEmpty
          className="osdOverviewPageFooter__button"
          flush="both"
          href={addBasePath('/app/management/opensearch-dashboards/settings#defaultRoute')}
          iconType="home"
          size="xs"
        >
          { i18n.translate(
            "opensearch-dashboards-react.pageFooter.changeHomeRouteLink".
            {
               defaultMessage: "Display a different page on log in"
            }
          )}
        </EuiButtonEmpty>
      </RedirectAppLinks>
    ) : (
...
```

What do you think, Josh?